src/activity_actor.cpp

```diff
@@ -4184,10 +4184,9 @@ void insert_item_activity_actor::finish( player_activity &act, Character &who )
             int charges = std::min( holstered_item.second, holster.max_charges_by_parent_recursive( it ) );
 
             if( charges > 0 && holster->can_contain_partial_directly( it ) ) {
-                int result = holster->fill_with( it, charges,
-                                                 /*unseal_pockets=*/true,
-                                                 /*allow_sealed=*/true,
-                                                 /*ignore_settings*/true );
+                const bool into_bottom = holstered_item.first.parent_item() &&
+                                         holster == holstered_item.first.parent_item().parent_item();
+                int result = holster->fill_with( it, charges, true, true, true, into_bottom );
```

Comment:
Ah, good to know the reasoning behind those comments, I'll re-add them.

Reply:
IDEs can expand those so they're not really needed

this is from clangd in vscode, after removing the comments
![Screenshot from 2023-07-20 19-17-01](https://github.com/CleverRaven/Cataclysm-DDA/assets/68240139/b2421207-bec6-4d81-ab98-a4aa93c0bd96)

Reply:
Huh, didn't know IDEs could do that nowadays. But I still think it's important to have those information in the code itself in case someone is using a less smart IDE (or no IDE at all like me).